### PR TITLE
various fixes

### PR DESCRIPTION
**Pull request by pifopi**

### CODE REVIEW

File: Package_Pokemon.py [hunk 1]
```diff
@@ -117,7 +117,7 @@
             move = copy(move_list[dump[i][0]])
             moves.append(move)
             if ability == 'Skill Link' and 'Hits 2-5 times' in move.effect:
-                move.power *= 5/2.1575
+                move.power *= 5/2.2575
```
Comment (ercdndrs) [inline Package_Pokemon.py]: Looking again at the math on Bulbapedia (some of which is wrong), I think this line should be:
`move.power *= 3.167`

Comment (pifopi) [reply Package_Pokemon.py]: indeed ! and the other place should be changed as well. i'll change that

Comment (pifopi) [reply Package_Pokemon.py]: updated !

Comment (ercdndrs) [reply Package_Pokemon.py]: Actually IDK why I even bothered adding the check for Skill Link. I don't think any of the rentals get it.

Comment (pifopi) [reply Package_Pokemon.py]: indeed, found none there : https://www.serebii.net/swordshield/dynamaxadventurespokemon.shtml

**Pull request merged**